.github/workflows/ic-ui-kit-main.yml

```diff
@@ -76,12 +76,39 @@ jobs:
       - name: Visual Regression tests
         run: npm run test-visual:ci
 
+  ic-ui-kit-cypress-tests:
+    name: "Cypress Tests"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - uses: browser-actions/setup-chrome@latest
```

Comment:
With what was being discussed earlier with running in Electron, is it an issue the setup here? This one may be my lack of understanding. If relevant, this comment is relevant to all CI files